fix fd mask detection on os x

From: Stefan Kaes <skaes@railsexpress.de>

This was broken in 3215b27a9abd8de793cf517f32d8901fd421eb1c
---
 configure.ac |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index e915cb21b7..90ce91bf7c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2039,7 +2039,8 @@ AC_CHECK_TYPES([clockid_t], [], [], [@%:@ifdef HAVE_TIME_H
 @%:@endif])
 
 AC_CACHE_VAL([rb_cv_large_fd_select],
-    [AC_CHECK_TYPE(fd_mask, [rb_cv_large_fd_select=yes], [rb_cv_large_fd_select=no], [@%:@ifdef HAVE_SYS_SELECT_H
+    [AC_CHECK_TYPE(fd_mask, [rb_cv_large_fd_select=yes], [rb_cv_large_fd_select=no], [AC_INCLUDES_DEFAULT([])
+@%:@ifdef HAVE_SYS_SELECT_H
 @%:@ include <sys/select.h>
 @%:@endif])])
 AS_IF([test "$rb_cv_large_fd_select" = yes], [
